File: languages/fortran/easy_hello.f90
! Fortran — Easy Example: Deterministic Hello
!
! What: Prints a stable greeting and exits successfully.
! Where: HPC bootstrap, numerical CI smoke, and scientific kernels.
! When: Use when the operational boundary is long-lived numerical Fortran code.
! Why: Fortran remains the performance baseline for large scientific workloads.
! How: A single program unit with no external libraries.

program easy_hello
  implicit none
  print '(A)', '{"status":"VERIFIED","language":"fortran","message":"hello-tower"}'
end program easy_hello

File: languages/java/easy_hello.java
// Java — Easy Example: Deterministic Hello
//
// What: Prints a stable greeting and exits successfully.
// Where: JVM bootstrap, CI smoke, and portable service entrypoints.
// When: Use when the platform contract is the JVM itself.
// Why: Java remains the baseline runtime for enterprise and Android ecosystems.
// How: A single public class with a main method and no external dependencies.

public final class easy_hello {
    public static void main(String[] args) {
        System.out.println("{\"status\":\"VERIFIED\",\"language\":\"java\",\"message\":\"hello-tower\"}");
    }

    private easy_hello() {}
}
